etc/ATSCam/primary_hdu.header

```diff
@@ -30,12 +30,12 @@ OBSGEO-Y=          -5208470.95 / [m] Y-axis Geocentric coordinate
 OBSGEO-Z=          -3195172.08 / [m] Z-axis Geocentric coordinate
 
         ---- Pointing info, etc. ----
-RA      =                      / Telescope RA 
-DEC     =                      / Telescope DEC
-RASTART =                      / RA of Target at start
-DECSTART=                      / DEC of Target at start
-RAEND   =                      / RA of Target at end
-DECEND  =                      / DEC of Target at end
+RA      =                      / Telescope RA from pointing component
+DEC     =                      / Telescope DEC from pointing component
+RASTART =                      / RA from telelescope AZ/EL position at start
```

Comment:
"RA of telescope calculated from AZSTART and ELSTART"

Reply:
Changed to `"RA of telescope from AZSTART and ELSTART"` in order to fit the 80 char limit for the FITS card.